FitGoal/FitGoal/Setting/AccountInfoCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  NotificationsCell.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 6/5/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class AccountInfoCell: UITableViewCell {
+    
+    var cellTitle =  "" {
+        didSet {
+            cellTitleLabel.attributedText = cellTitle.formattedText(font: "Roboto-Light", size: 15, color: #colorLiteral(red: 0.5215686275, green: 0.5333333333, blue: 0.568627451, alpha: 1), kern: 0.3)
+            if cellTitle == "Notifications" {
```

Comment:
You shouldn't use the value of the title for this, try to use a specific property for that configuration. Once we add localisation for the app this code won't work anymore.